ds3/ds3.py

```diff
@@ -11336,10 +11340,10 @@ def get_service(self, request):
             raise TypeError('request for get_service should be of type GetServiceRequest but was ' + request.__class__.__name__)
         return GetServiceResponse(self.net_client.get_response(request), request)
     
-    def get_object(self, request):
+    def get_object(self, request, buffer_size=None):
```

Comment:
On second thought, we probably should set this to something like 1MB instead of letting it be unspecified.

Reply:
SI or Binary? ie 1,000,000 bytes or 1,048,576 bytes?

Reply:
Binary.